sdk/python/feast/type_map.py

```diff
@@ -353,6 +354,19 @@ def _python_value_to_proto_value(
                 feast_value_type
             ]
 
+            # Bytes to array type conversion
+            if isinstance(sample, (bytes, bytearray)):
+                # Bytes of an array containing elements of bytes not supported
+                if feast_value_type == ValueType.BYTES_LIST:
+                    raise _type_err(sample, ValueType.BYTES_LIST)
+
+                json_value = json.loads(sample)
+                if isinstance(json_value, list):
+                    if feast_value_type == ValueType.BOOL_LIST:
+                        json_value = [bool(item) for item in json_value]
+                    return [ProtoValue(**{field_name: proto_type(val=json_value)})]
+                raise _type_err(sample, valid_types[0])
```

Comment:
for raw data that's bytes but an feature defined as a list type, attempt a json loads